src/libraries/System.IO.FileSystem.Watcher/tests/Utility/FileSystemWatcherTest.cs

```diff
@@ -40,26 +40,17 @@ public static (AutoResetEvent EventOccured, FileSystemEventHandler Handler) Watc
             FileSystemEventHandler changeHandler = (o, e) =>
             {
                 Assert.Equal(WatcherChangeTypes.Changed, e.ChangeType);
-                VerifyExpectedPaths(expectedPaths, e);
-                eventOccurred.Set();
+
+                if (expectedPaths == null || expectedPaths.Contains(e.FullPath))
```

Comment:
so far, the `VerifyExpectedPaths` was using `Path.GetFullPath(e.FullPath)`, should we keep doing that?

```suggestion
                if (expectedPaths == null || expectedPaths.Contains(Path.GetFullPath(e.FullPath)))
```

Reply:
We changed FileSystemEventArgs.FullPath this release to actually return the full path https://github.com/dotnet/runtime/issues/62606, so we shouldn't need GetFullPath anymore.